common/djangoapps/django_comment_common/migrations/0004_auto_20161108_1127.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
I think someone committed model changes without a migration, and I ran make migrations to get my devstack running again, then inadvertently committed the file.